front-end/src/components/VPLink/VPLinkModel.js

```diff
@@ -25,11 +31,9 @@ export class VPLinkModel extends DefaultLinkModel {
      * TODO: Notify backend the link has been removed
     */
     remove() {
-      const sourcePort = this.getSourcePort(); // PortModel
-      const sourceNode = sourcePort.getNode(); // NodeModel
-      const targetPort = this.getTargetPort(); // PortModel
-      const targetNode = targetPort.getNode(); // NodeModel
-      
-      super.remove();
```

Comment:
It's not the best solution, but if we're looking for a quick-fix for demo-purposes, moving the `super.remove()` before the API call will visually delete the edge without needing to click again.

Reply:
Good call, I'm going to go with that for now.